packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -409,12 +409,15 @@ const schema: SchemaType<"Posts"> = {
     order: 10,
     group: formGroups.adminOptions,
     onCreate: ({document: post}) => {
-      if(!post.sticky) {
+      if(!isEAForum && !post.sticky) {
         return false;
       }
     },
     onUpdate: ({modifier}) => {
-      if (!modifier.$set.sticky) {
+      // WH 2025-03-17: I think this is a bug in general, as a non-admin editing this post will cause
+      // sticky to be set to false. Forum-gating to EAF to speed up fixing this live bug for us, but
+      // I believe this function and `onCreate`
+      if (!isEAForum && !modifier.$set.sticky) {
```

Comment:
@jimrandomh @b0b3rt I don't know of any reason why this can't be removed for everyone, but it was last edited 6 years ago so I didn't want to risk it seeing as we want to get this out today. Let me know if you want me to make a followup to remove this entirely (and for stickyMeta and afSticky)

Reply:
Huh, weird.  That does look like a bug but I'm confused by why that's triggered on the EA forum but not on LW.  Some server-side updateMutator that only the EA forum uses, maybe?

Reply:
Doesn't sound right but not thinking of many other great ideas.